packages/caver-account/src/accountKey/accountKeyHelper.js

```diff
@@ -25,6 +28,65 @@ const ACCOUNT_KEY_TAG = {
     ACCOUNT_KEY_ROLE_BASED_TAG: '0x05',
 }
 
+/**
+ * Creates and returns the valid instance of WeightedMultiSigOptions for AccountKeyWeightedMultiSig.
+ * If the user does not define the values of options(threshold, weights),
+ * default options(threshold is 1 and the weight of each key is 1) are returned.
+ *
+ * @param {number} lengthOfKeys The lenght of keys.
+ * @param {WeightedMultiSigOptions|object} [options] An instance of WeightedMultiSigOptions or an object that defines 'threshold' and 'weight'.
+ * @return {WeightedMultiSigOptions}
+ */
+const fillWeightedMultiSigOptionsForMultiSig = (lengthOfKeys, options) => {
```

Comment:
Do we need to get `lengthOfKeys`? What if the user passes different values of lengthOfKeys and options.weights.length?

Reply:
This function, user cannot access directly, so we don't have to worry about passing invalid legnthOfKeys.
But it is still possible for the user to pass invalid weight array. 
But that validation is done in AccountKeyWeightedMultiSig class.
There is no validation for lengthOfKeys as this function is intended to create a default option if no option is defined.

Reply:
How about passing PublicKeys array instead lengthOfKeys?

Reply:
I originally implemented with publickey array, but in this function required information is only length, so i chagned to length. Do you think passing publicKey array is better?

Reply:
I think it can be reduced user error by passing the entire public key array and determining the length inside the function.

Reply:
What I thought was better to pass the key length in the code becuase user can't use this function directly. This is only for internal use.

Reply:
got it. I think it's okay if it is used internally. :)